.github/actions/collect-troubleshooting/action.yml

```diff
@@ -26,6 +26,10 @@ runs:
       run: kubectl describe pod -l kmm.node.kubernetes.io/module.name
       shell: bash
 
+    - name: Describe Jobs
+      run: kubectl describe job.batch -l kmm.node.kubernetes.io/module.name
+      shell: bash
```

Comment:
I don't think this will work. I think that the GC will remove it before we reach this stage.
This is why we have https://github.com/kubernetes-sigs/kernel-module-management/blob/107d0b054ef6d5a66fc07c5a321fdddc75f7353c/ci/prow/e2e-incluster-build#L48.